src/Jackett.Common/Definitions/yggtorrent.yml

```diff
@@ -170,8 +170,8 @@ settings:
 
 login:
   method: form
-  path: /
-  form: "#user-login"
+  path: /auth/login
+  form: form[action="/auth/process_login"]
   inputs:
     id: "{{ .Config.username }}"
     pass: "{{ .Config.password }}"
```

Comment:
please remove ci_csfr_token as it is no longer required for login
also replace the error block and the test block with this new one
```yaml
  error:
    - selector: p.error-form
  test:
    path: /
    selector: a[href$="/user/logout"]
```

Reply:
done 👍